awscrt/mqtt.py

```diff
@@ -440,26 +634,33 @@ class WebsocketHandshakeTransformArgs(object):
     """
     Argument to a "websocket_handshake_transform" function.
 
-    -- Attributes --
-    mqtt_connection: awscrt.mqtt.Connection this handshake is for.
-    http_request: awscrt.http.HttpRequest for this handshake.
-
     A websocket_handshake_transform function has signature:
-    (WebsocketHandshakeTransformArgs) -> None
+    ``fn(transform_args: WebsocketHandshakeTransformArgs, **kwargs) -> None``
 
-    The function implementer may modify transform_args.http_request as desired.
-    They MUST call transform_args.set_done() when complete, passing an
-    exception if something went wrong. Failure to call set_done()
+    The function implementer may modify `transform_args.http_request` as desired.
+    They MUST call `transform_args.set_done()` when complete, passing an
+    exception if something went wrong. Failure to call `set_done()`
     will hang the application.
 
-    The implementor may do asynchronous work before calling transform_args.set_done(),
-    they are not required to call set_done() within the scope of the transform function.
+    The implementor may do asynchronous work before calling `transform_args.set_done()`,
```

Comment:
Implementer. Implementor is the powerful antagonist in a programmer-centric comic book

Reply:
YOU MAKE IMPLEMENTOR ANGRY